src/scraper.rs

```diff
@@ -136,12 +136,8 @@ pub struct MeetingTime {
 #[derive(Debug, Clone)]
 pub struct Section {
     pub crn: u64,
-    pub subject_code: String,
-    pub course_code: String,
```

Comment:
these Section fields are still present in the DB, no? I think that we should make `Section` always exactly match the corresponding `sections` table schema

Reply:
Fair enough, that sounds reasonable.

My take on this was that `subject_code` and `course_code` are just foreign keys to populate the `Course.sections` Vec, similarly to how `MeetingTime` doesn't have the `crn`, but I suppose it makes sense to duplicate the info for display or backlinking purposes.

Reply:
Done, I readded the fields and rebased the commits :+1: